core/src/main/kotlin/keiyoushi/utils/Date.kt

```diff
@@ -3,12 +3,11 @@ package keiyoushi.utils
 import java.text.ParseException
 import java.text.SimpleDateFormat
 
-@Suppress("NOTHING_TO_INLINE")
-inline fun SimpleDateFormat.tryParse(date: String?): Long {
+fun SimpleDateFormat.tryParse(date: String?): Long {
     date ?: return 0L
 
     return try {
-        parse(date)?.time ?: 0L
+        parse(date)!!.time
```

Comment:
Originally mentioned at https://github.com/keiyoushi/extensions-source/pull/1808#discussion_r1521311791:

> It won't be `null` at all, see [`Date parse(String) throws ParseException`](https://docs.oracle.com/javase/8/docs/api/java/text/DateFormat.html#parse-java.lang.String-). IDE wrongly prompts it as nullable because it wraps [`Date? parse(String, ParsePosition)`](https://docs.oracle.com/javase/8/docs/api/java/text/DateFormat.html#parse-java.lang.String-java.text.ParsePosition-) and throws on a check of `ParsePosition` so the IDE can't tell.